integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/CoinbaseBuyDashOrderReviewFragment.kt

```diff
@@ -63,7 +58,6 @@ class CoinbaseBuyDashOrderReviewFragment : Fragment(R.layout.fragment_coinbase_b
     private lateinit var selectedPaymentMethodId: String
     private var loadingDialog: FancyAlertDialog? = null
     private var isRetrying = false
-    private var transactionStateDialog: CoinBaseBuyDashDialog? = null
     private var newBuyOrderId: String? = null
     @Inject
     lateinit var analyticsService: FirebaseAnalyticsServiceImpl
```

Comment:
Not the current change, but we should be injecting the interface and not the implementation.